### PR TITLE
Update expected AppVeyor env variables.

### PR DESCRIPTION
**Pull request by daspecster**
Add `APPVEYOR_PULL_REQUEST_NUMBER` and `APPVEYOR_PULL_REQUEST_TITLE` to expected environment variables.

### CODE REVIEW

**Comment by dhermes** [comment]
@daspecster WDYT about making this a real PR we can merge by fixing the AppVeyor ["breakage"][2] and adding `APPVEYOR_PULL_REQUEST_NUMBER` and `APPVEYOR_PULL_REQUEST_TITLE` to the [`EXPECTED`][1] list

(I made these scripts for exactly this reason, to detect new features from the services.)

[1]: https://github.com/dhermes/ci-diff-helper/blob/84b14d000b75a47592116ff1a99dfcd38614dae3/appveyor_environ.py#L18
[2]: https://ci.appveyor.com/project/dhermes/ci-diff-helper/build/1.0.197.test-app

**Comment by daspecster** [comment]
Sure thing!
https://github.com/dhermes/ci-diff-helper/pull/13/commits/32eb579b1c8e982fe5618bd600a85ca3319bc6d6

File: appveyor_environ.py [hunk 1]
```diff
@@ -28,6 +28,8 @@
     'APPVEYOR_PROJECT_ID',
     'APPVEYOR_PROJECT_NAME',
     'APPVEYOR_PROJECT_SLUG',
+    'APPVEYOR_PULL_REQUEST_NUMBER',
+    'APPVEYOR_PULL_REQUEST_TITLE',
```
Comment (dhermes) [inline appveyor_environ.py]: I'm surprised there are no existing vars. beginning with `APPVEYOR_PULL*`. Very strange AppVeyor!

File: circle_ci_environ.py [hunk 2]
```diff
@@ -41,6 +41,7 @@
     'CI_PULL_REQUEST',
     'CI_PULL_REQUESTS',
     'CI_REPORTS',
+    'YOLO_ENV',
```
Comment (dhermes) [inline circle_ci_environ.py]: @daspecster Remove this, squash into a single commit and then update the PR name from "Yolo environment" so we can merge?

Comment (daspecster) [inline circle_ci_environ.py]: Done and done.

**Pull request merged**